lib/src/material_auto_suggest_input_demo/material_auto_suggest_input_demo.html

```diff
@@ -0,0 +1,137 @@
+        [shouldClearOnSelection]="shouldClearOnSelection"
+        [label]="label"
+        [labelRenderer]="labelRenderer"
+        [leadingGlyph]="leadingGlyph"
+        [emptyPlaceholder]="emptyPlaceholder"
+        [selection]="singleModel"
+        [selectionOptions]="options"
+        [itemRenderer]="itemRenderer"
+        [componentRenderer]="componentRenderer"
+        [popupPositions]="popupPositions"
+        [limit]="limit"
+        [disabled]="disabled">
+    </material-auto-suggest-input>
+
+    <div>Model's selected value:</div>
+    <div *ngFor="let v of singleModel.selectedValues">{{v}}</div>
+    <br><br><br>
+
+    <h2>Multi selection</h2>
+    <material-auto-suggest-input
+        [(inputText)]="inputText"
+        [filterSuggestions]="filterSuggestions"
+        [popupMatchInputWidth]="popupMatchInputWidth"
+        [showClearIcon]="showClearIcon"
+        [hideCheckbox]="hideCheckbox"
+        [shouldClearOnSelection]="shouldClearOnSelection"
+        [label]="label"
+        [labelRenderer]="labelRenderer"
+        [leadingGlyph]="leadingGlyph"
+        [emptyPlaceholder]="emptyPlaceholder"
+        [selection]="multiModel"
+        [selectionOptions]="options"
+        [itemRenderer]="itemRenderer"
+        [componentRenderer]="componentRenderer"
+        [popupPositions]="popupPositions"
+        [limit]="limit"
+        [disabled]="disabled">
+    </material-auto-suggest-input>
+
+    <div>Model's selected values:</div>
+    <div *ngFor="let v of multiModel.selectedValues">{{v}}</div>
+  </div>
+  <div class="options-pane">
+    <h3>Options</h3>
+    <material-checkbox [(checked)]="useLabelRenderer">
+      Use label renderer
+    </material-checkbox>
+    <br>
+    <material-checkbox [(checked)]="useComponentRenderer">
+      Use component renderer
+    </material-checkbox>
+    <br>
+    <material-checkbox [(checked)]="filterSuggestions">
+      Filter suggestions
+    </material-checkbox>
+    <br>
+    <material-checkbox [(checked)]="shouldClearOnSelection">
+      Clear on selection
+    </material-checkbox>
+    <br>
+    <material-checkbox [(checked)]="popupMatchInputWidth">
+      Popup match input width
+    </material-checkbox>
+    <br>
+    <material-checkbox [(checked)]="showClearIcon">
+      Show clear icon
+    </material-checkbox>
+    <br>
+    <material-checkbox [(checked)]="hideCheckbox">
+      Hide checkbox for multi selection
+    </material-checkbox>
+    <material-checkbox [(checked)]="disabled">
+      Disable input
+    </material-checkbox>
+    <br>
+    <material-input
+        floatingLabel label="Label"
+        [(ngModel)]="label"
+        [displayBottomPanel]="false">
+    </material-input>
+    <material-input
+        floatingLabel label="Empty placeholder"
+        [(ngModel)]="emptyPlaceholder"
+        [displayBottomPanel]="false">
+    </material-input>
+    <material-input
+        floatingLabel label="Leading glyph"
+        [(ngModel)]="leadingGlyph"
```

Comment:
I wish this was a search box – unclear what options are available

Reply:
I'll be working on updating the examples soon.